src/assets/locales/locale_config.js

```diff
@@ -4,11 +4,13 @@
 
 export const AVAILABLE_LOCALES = {
   en: "English",
-  zh: "简体中文"
+  zh: "简体中文",
+  pt: "Portugês"
```

Comment:
The way this is configured right now it's not finding the locale correctly as it willl look for **pt.json** localization file. For the fallback mechanism to correctly work when the browser language is set to **Português (Brasil)** we need the available locale to be set to **pt-br**:

```
export const AVAILABLE_LOCALES = {
  en: "English",
  zh: "简体中文",
  "pt-br": "Portugês (Brasil)"
};
```

And fallback to **Português** until we have a **Português** localization:
```
export const FALLBACK_LOCALES = {
  "zh-cn": "zh",
  "zh-hans": "zh",
  "zh-hans-cn": "zh",
  pt: "pt-br"
};
```

Reply:
@johnshaughnessy Can you please confirm this?

Reply:
Yes that's correct and the subsequent commit fixed this.

(Oops, I missed this ping. My github notifications are all messed up / disabled at the moment. I'll try to fix them so I don't miss things but also feel free to dm me on discord)